File: July 2022/July_19.java
class Solution {
    public List<List<Integer>> generate(int numRows) {
        List<List<Integer>> list = new ArrayList<>();
        for(int i = 1; i <= numRows; i++){
            List<Integer> al = new ArrayList<>();
            al.add(1);
            for(int j = 2; j < i; j++)
                al.add(list.get(i - 2).get(j - 2) + list.get(i - 2).get(j - 1));
            if(i > 1)
                al.add(1);
            list.add(al);
        }
        return list;
    }
}

File: July 2022/July_19.cpp
class Solution {
public:
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>> vec;
        for(int i = 1; i <= numRows; i++){
            vector<int> v;
            v.push_back(1);
            for(int j = 2; j < i; j++)
                v.push_back(vec[i - 2][j - 2] + vec[i - 2][j - 1]);
            if(i > 1)
                v.push_back(1);
            vec.push_back(v);
        }
        return vec;
    }
};
